modules/gh-oidc/README.md

```diff
@@ -0,0 +1,111 @@
+## GitHub OIDC
+
+This module handles the opinionated creation of infrastructure necessary to configure [Workload Identity pools](https://cloud.google.com/iam/docs/workload-identity-federation#pools) and [providers](https://cloud.google.com/iam/docs/workload-identity-federation#providers) for authenticating to GCP using GitHub Actions OIDC tokens.
+
+This includes:
+
+- Creation of a Workload Identity pool
+- Configuring a Workload Identity provider
+- Granting external identities necessary IAM roles on Service Accounts
+
+### Example Usage
+
+```terraform
+module "gh_oidc" {
+  source      = "terraform-google-modules/github-actions-runners/google//modules/gh-oidc"
+  project_id  = var.project_id
+  pool_id     = "example-pool"
+  provider_id = "example-gh-provider"
+  sa_mapping = {
+    "foo-service-account" = {
+      sa_name   = "foo-service-account@my-project.iam.gserviceaccount.com"
+      attribute = "attribute.repository/user/repo"
```

Comment:
It might be helpful to clarify what this `attribute` refers to by linking to the appropriate docs somewhere.

Reply:
I'll update in a follow up once we have official docs.